### ISSUE TITLE
TabCompleteEvent not being called

### ISSUE THREAD

**Issue by ShaneBeee** [PaperSpigot/Paper#10833]
### Expected behavior

for the event to be called

### Observed/Actual behavior

I noticed after the big command API update, the TabCompleteEvent is no longer being called

### Steps/models to reproduce

just some simple code
```java
@EventHandler
public void onTabComplete(TabCompleteEvent event) {
    Bukkit.broadcastMessage("tab complete");
}
```

### Plugin and Datapack List

[13:14:09 INFO]: Server Plugins (10):
[13:14:09 INFO]: Bukkit Plugins:
[13:14:09 INFO]:  - BeeConomy, PermissionsEx, RandomTeleport, SkBee, SkBriggy, Skript, skript-reflect, spark, StressTestBots, Vault
> 

### Paper version
```
[13:25:12 INFO]: Checking version, please wait...
[13:25:12 INFO]: This server is running Paper version 1.20.6-126-master@06e69c8 (2024-05-30T18:38:45Z) (Implementing API version 1.20.6-R0.1-SNAPSHOT)
You are running the latest version
Previous version: 1.20.6-124-0fcf3e3 (MC: 1.20.6)
```

**Comment by Machine-Maker** [comment]
In a few minutes, a test build will be available at the top of https://github.com/PaperMC/Paper/pull/10834. Does that fix your issue?

**Comment by ShaneBeee** [comment]
> In a few minutes, a test build will be available at the top of #10834. Does that fix your issue?

just tested, works splendidly, thank you 👍🏻
